src/channels.c

```diff
@@ -743,20 +746,21 @@ channel_access(channel_t *ch, access_t *a, int disabled)
   }
 
   /* Channel tag check */
+  if (a->aa_chtags_exclude) {
+    HTSMSG_FOREACH(f, a->aa_chtags_exclude)
+      LIST_FOREACH(ilm, &ch->ch_ctms, ilm_in2_link)
+        if (!strcmp(htsmsg_field_get_str(f) ?: "",
```

Comment:
Here seems to be a clash, because the channel can be in both excluded / allowed tag lists (if the channel belongs to the multiple lists). But it's probably OK to check the excluded conditions as first.